ev3dev/core.py

```diff
@@ -955,6 +958,94 @@ def wait_while(self, s, timeout=None):
         """
         return self.wait(lambda state: s not in state, timeout)
 
+    def set_power(self, power):
+        self.speed_sp = power
+
+    def set_position_rotations(self, power, rotations):
+        if power > 0:
+            self.position_sp = self.position + int(rotations * self.count_per_rot)
+        else:
+            self.position_sp = self.position - int(rotations * self.count_per_rot)
+
+    def set_position_degrees(self, power, degrees):
+        rotations = float(degrees / self.count_per_rot)
+        self.set_position_rotations(power, rotations)
+
+    def set_brake(self, brake):
+        if brake:
+            self.stop_action = self.STOP_ACTION_BRAKE
+        else:
+            self.stop_action = self.STOP_ACTION_COAST
+
+    def set_time(self, ms):
```

Comment:
Function calls in python are expensive. Here and in several other places, your are just setting an attribute. It would be more efficient to just set the attribute directly.

Reply:
ack will chop `set_power()` and `set_time()`